plugin/keyring_vault/tests/mtr/mount_point_service.inc

```diff
@@ -0,0 +1,62 @@
+# Creates or deletes secret mount point specified in keyring_vault configuration file
+# Mount point is :
+#  - created when MOUNT_POINT_SERVICE_OP is set to 1
```

Comment:
Why not use "CREATE", "DELETE" or some other descriptive strings instead of 1 and 2?

Reply:
The problem is where to define those CREATE and DELETE. Test includes mount_point_service.inc and it needs to define what operation it wants mount_point_service to perform. The perfect solution would be --let MOUNT_POINT_SERVICE_OP=CREATE and then source mount_point_service.inc, but the CREATE would need to be defined before this source operation and in every test that source mount_point_service.inc

Reply:
Not CREATE but "CREATE" ?

Reply:
ahh, right, changing

Reply:
Done